docs/pages/guides/authentication.md

```diff
@@ -2314,7 +2314,7 @@ https://auth.expo.io/@johnsample/my-app
 
 - **Environment:** Development or production projects in Expo Go, or in a standalone build.
 - **Create:** Use `AuthSession.makeRedirectUri({ useProxy: true })` to create this URI.
-  - The link is constructed from your Expo username and the Expo app name, which are appended to the proxy website.
+  - The link is constructed from your Expo username and the app's EAS Update `slug`, which are appended to the proxy website.
```

Comment:
'slug' isn't something specific to EAS Update, in fact the opposite.

Reply:
```suggestion
  - The link is constructed from your Expo username and the Expo config (**app.json**) `slug`, which are appended to the proxy website.
```